### ISSUE TITLE
try to do experiment on 70-classes dataset and the model doesn't converge

### ISSUE THREAD

**Issue by yyht** [guoyinwang/LEAM#11]
hi, i am very interested in your paper. I tried to do experiment on my own Finance News dataset to predict finicial event type given the finicial news but, the model doesn't converge that the loss on training set and development set are stuck on a value and change slowly. I am looking forward to your reply for deeper discussion. Thanks

**Comment by greeness** [comment]
Try to reduce your learning rate by 10x, 100x and see if training loss decreases.
